README.md

```diff
@@ -783,31 +783,18 @@ This is a known issue:
 
 ### vim.call()
 
-`vim.call()` calls a Vimscript function. This can either be a built-in Vim function or a user function. Again, data types are converted back and forth from Lua to Vimscript.
+`vim.call()` is the less readable choice to call a Vimscript function. It should be avoided in favour of the equivalent `vim.fn.{function}()`.
+The Vimscript function can either be a built-in Vim function or a user function. Data types are converted back and forth from Lua to Vimscript.
 
-It takes in the name of the function followed by the arguments you want to pass to that function:
-
-```lua
-print(vim.call('printf', 'Hello from %s', 'Lua'))
-
-local reversed_list = vim.call('reverse', { 'a', 'b', 'c' })
-print(vim.inspect(reversed_list)) -- { "c", "b", "a" }
-
-local function print_stdout(chan_id, data, name)
-    print(data[1])
-end
-
-vim.call('jobstart', 'ls', { on_stdout = print_stdout })
-
-vim.call('my#autoload#function')
-```
+Function names can be converted to `vim.fn.{function}()` by 1.using `vim.fn.printf(...)` instead of `vim.call(printf,..)` and
+2.for autoload functions using `vim.fn['my#autoload#function']()` instead of `vim.call('my#autoload#function')`.
 
 See also:
 - `:help vim.call()`
 
 ### vim.fn.{function}()
 
-`vim.fn` does the exact same thing as `vim.call()`, but looks more like a native Lua function call:
+`vim.fn` does internally use `vim.call()`. It is the recommended choice, as it looks more like a native Lua function call:
```

Comment:
Do you have a source for it being the recommended choice? I don't see anything in the docs

Reply:
word count for vim.fn 90, word count for vim.call 18, but thats probably a bit strong of a formulation.
What do you think on "This guide recommends it, as it separates function name and arguments more explicitly:" ?

Reply:
I'd rather keep the guide mostly descriptive rather than prescriptive. IMO the simplest change here is to just drop the `vim.call` section and keep only `vim.fn`.

Reply:
@nanotee I dont quite understand. So you want to drop the one section without explanation?

Reply:
I'm not sure I understand either. I'm totally okay with removing clutter like you suggested, my only concern here is that there might be valid use cases for `vim.call` that I'm not aware of, so I don't want to say "prefer `vim.fn` over `vim.call`". Sorry if I miscommunicated

Reply:
@nanotee Should I add a PR with RFC to the help page for clarification? Or how should we proceed on this?

Reply:
If you feel like that's a point of confusion, then it's probably worth asking the core devs for their opinion on Gitter or in a Github Discussions thread